analysis/src/ProcessCmt.ml

```diff
@@ -99,6 +106,12 @@ let rec forTypeSignatureItem ~(env : SharedTypes.Env.t) ~(exported : Exported.t)
                            optional =
                              Res_parsetree_viewer.hasOptionalAttribute
                                ld_attributes;
+                           docstring =
+                             (match
+                                ProcessAttributes.findDocAttribute ld_attributes
+                              with
+                             | None -> []
+                             | Some docstring -> [docstring]);
```

Comment:
This code appears many times -- how about refactor into a separate function.

Reply:
Isn't there such a function already? If not, the existing places creating docstrings should use it too.